requirements.txt

```diff
@@ -21,5 +21,6 @@ retry==0.9.1
 setuptools==39.2.0
 stomp.py==4.1.15
 rucio-clients==1.19.3
+CMSMonitoring>=0.1.1
```

Comment:
From another thread, I've seen this version is broken. But don't bother about updating it, there might be other things to be fixed when pushing it in and then I'll just create a new PR.